packages/video_player/video_player_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.3.12
+
+* Adds option to configure the User-Agent in the HTTP headers.
```

Comment:
There should be a second list item here similar to what you added to the app-facing package, since there is a behavioral change in sending headers in general.